django/test/selenium.py

```diff
@@ -133,3 +137,86 @@ def disable_implicit_wait(self):
             yield
         finally:
             self.selenium.implicitly_wait(self.implicit_wait)
+
+
+def screenshot_cases(method_names):
+    if isinstance(method_names, str):
+        method_names = method_names.split(",")
+
+    def wrapper(func):
+        func._screenshot_cases = method_names
+        return func
+
+    return wrapper
+
+
+@tag("screenshot")
+class ScreenshotTestCase(SeleniumTestCase):
+    screenshots = False
+
+    @classmethod
+    def __init_subclass__(cls, **kwargs):
+        super().__init_subclass__(**kwargs)
+
+        for name, func in list(cls.__dict__.items()):
+            if not cls.screenshots:
+                continue
+            if not isinstance(func, FunctionType):
+                continue
```

Comment:
I suggest deleting this since the attribute can come only from the decorator
```suggestion
```

I know you copied it from unittest-parametrize but that needs to be a bit more defensive because of the chance of interaction with other code. Whilst I'd say don't add code here that's not necessary.